aws-globalaccelerator-endpointgroup/src/main/kotlin/software/amazon/globalaccelerator/endpointgroup/UpdateHandler.kt

```diff
@@ -24,13 +24,10 @@ class UpdateHandler : BaseHandler<CallbackContext>() {
         val inferredCallbackContext = callbackContext
                 ?: CallbackContext(stabilizationRetriesRemaining = HandlerCommons.NUMBER_OF_STATE_POLL_RETRIES, pendingStabilization = false)
         val model = request.desiredResourceState
-        HandlerCommons.getEndpointGroup(model.endpointGroupArn, proxy, agaClient, logger)
-                ?: return ProgressEvent.defaultFailureHandler(
-                        Exception("Failed to find endpoint group with arn:[${model.endpointGroupArn}]"),
-                        HandlerErrorCode.NotFound
-                )
         return if (!inferredCallbackContext.pendingStabilization) {
-            updateEndpointGroup(model, request.previousResourceState, proxy, agaClient, logger)
+            HandlerCommons.getEndpointGroup(model.endpointGroupArn, proxy, agaClient, logger)
```

Comment:
Can you please confirm if it matches other customer facing messages?

Reply:
Yes, patterned after other message in the code.